### PR TITLE
Feature/navigation

### PR DESCRIPTION
**Pull request by Steffen-Anslinger**

### CODE REVIEW

File: components/Navigation/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  color: black;
+
+  &:hover {
+    text-decoration: underline red;
+    background-color: lightgray;
+  }
+`;
+
+const StyledNav = styled.nav`
+  align-items: center;
```
Comment (CJAkkuec) [inline components/Navigation/index.js:16]:
```suggestion
```
Pretty sure this does nothing. 🤔

File: pages/create.js [hunk 2]
```diff
@@ -1,5 +1,11 @@
+import Navigation from "@/components/Navigation";
 import ProjectForm from "@/components/ProjectForm";
 
 export default function CreatePage({ onAddProject, projects }) {
-  return <ProjectForm onAddProject={onAddProject} projects={projects} />;
+  return (
+    <div>
+      <ProjectForm onAddProject={onAddProject} projects={projects} />;
+      <Navigation />;
+    </div>
```
Comment (CJAkkuec) [inline pages/create.js:9]:
```suggestion
    <>
      <ProjectForm onAddProject={onAddProject} projects={projects} />;
      <Navigation />;
    </>
```
If you only want to appease the JSX gods, the fragments are enough.

File: pages/index.js [hunk 3]
```diff
@@ -2,22 +2,23 @@ import Header from "@/components/Header";
...
+  background-color: lightgray;
+  color: black;
+  padding: 1em 1.5em;
+  text-decoration: none;
+  border-radius: 5px;
+`;
 
 export default function HomePage({ projects }) {
   return (
     <div>
```
Comment (CJAkkuec) [inline pages/index.js:17]:
```suggestion
    <>
```

File: pages/create.js [hunk 4]
```diff
@@ -1,5 +1,11 @@
+import Navigation from "@/components/Navigation";
 import ProjectForm from "@/components/ProjectForm";
 
 export default function CreatePage({ onAddProject, projects }) {
-  return <ProjectForm onAddProject={onAddProject} projects={projects} />;
+  return (
+    <div>
```
Comment (moonwave99) [inline pages/create.js:6]: Here we can spare the `<div>` and use a fragment - `<>...</>`

File: pages/index.js [hunk 5]
```diff
@@ -2,22 +2,23 @@ import Header from "@/components/Header";
...
       <Header />
+
       <Projects projects={projects} />
-      <StyledLink href="/create">Create new project</StyledLink>
+      <Navigation />
     </div>
```
Comment (CJAkkuec) [inline pages/index.js:22]:
```suggestion
    </>
```

File: pages/index.js (hunk 3 above)
Comment (moonwave99) [inline pages/index.js:17]: See above

**Review by CJAkkuec — APPROVED** [review]
Looking good! 🚀 

The changes are so minimal that I trust you to implement them. No updated PR to be posted again necessary.

**Review by moonwave99 — CHANGES_REQUESTED** [review]

**Pull request merged**